hubblestack/extmods/returners/splunk_nova_return.py

```diff
@@ -169,6 +169,12 @@ def _generate_event(args, cloud_details, custom_fields, check_type=None, data=No
             for key, value in data[args['check_id']].items():
                 if key not in ['tag']:
                     event[key] = value
+    if check_type == 'Failure':
+        detail = data['detail']
+        event.update({'cve': detail['cve'],
```

Comment:
Please add validation checks here if detail has keys like 'cve', 'advisory', 'severity' and 'installed', then only it should update the event to be published.

Reply:
@devagupt I have added this.  Please let me know if this is what you're looking for.

Reply:
Although this looks fine to me, but I am more inclined to have a separate returner for oval module. 
As new changes in nova are planned in near future, there are considerable changes in nova schema and parsing the output from a module in returner will only add maintainability costs.

Reply:
@devagupt I can agree with that, but I don't see why we should hold up implementing improvements to the oval_scanner at this current time.  We can later refactor the returner logic into a separate module if it seems feasible.

Reply:
@buddwm Sure, we should pick up this task in new nova related release.